GetStream/stream-chat-android
# Jump to newest message with refrsh

### 🎯 Goal

Draft for fix of gap created after clicking in bottom to go to first messages.

### 🛠 Implementation details

Allow request to have information about the need of messages to be refreshed.

### 🎨 UI Changes

<table>
<thead>
<tr>
<th>Before</th>
<th>After</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<video src="https://user-images.githubusercontent.com/10619102/179286964-3e26331d-9c30-417f-8c74-3b5ce5046b65.mov" controls="controls" muted="muted" />
</td>
<td>
<video src="https://user-images.githubusercontent.com/10619102/179286997-22df2168-d516-44a8-89fb-f11d27611fa8.mov" controls="controls" muted="muted" />
</td>
</tr>
</tbody>
</table>
### 🧪 Testing

On develop:
- Search for a message 
- Click in the buttom to go to bottom
- Check that a gap in messages are created

Now move to this branch and check that the problem is fixed.


### ☑️Contributor Checklist

#### General
- ~[ ] PR is linked to the GitHub issue it resolves~ This fixes something now published yet.

#### Code & documentation
- ~[ ] Changelog is updated with client-facing changes~.  This fixes something now published yet. 

### ☑️Reviewer Checklist

### 🎉 GIF
